deploy/pom.xml

```diff
@@ -660,6 +665,16 @@
             <artifactId>commons-collections4</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-configuration2</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+          <scope>compile</scope>
+        </dependency>
```

Comment:
commons-logging? I thought we got rid of this one

Reply:
It's included as a dependency for commons-configuration2.
https://commons.apache.org/proper/commons-configuration/dependencies.html

Reply:
re-introducing an old dependency that was removed is prob a bad idea. if commons-configuration is already being used, just use that version ;)

Reply:
Agree with Sam, I had the same concern, let's review this Markus.

Reply:
I could remove commons-logging as a dependency for commons-configuration2 without seeing side effects so far.
Currently commons-configuration (1) is used in delegatedaccess, rubrics and help and it's not in the shared libs.
As for commons-configuration2 it is used already aswell in basiclti. I would perfer to stick with the newer version, as it's used already and commons-configuration does not seem to have the best history with plist support.